src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx

```diff
@@ -1,16 +1,33 @@
 import {Link} from 'react-router';
 import React from 'react';
-import PropTypes from 'prop-types';
 import Cookies from 'js-cookie';
 import styled from 'react-emotion';
 
-import {t, tct} from 'app/locale';
+import {t} from 'app/locale';
 import InlineSvg from 'app/components/inlineSvg';
-import SentryTypes from 'app/sentryTypes';
-import replaceRouterParams from 'app/utils/replaceRouterParams';
-import withLatestContext from 'app/utils/withLatestContext';
 import space from 'app/styles/space';
 
+class SettingsBackButton extends React.Component {
+  render() {
+    // if the user needs to setup 2fa as part of the org invite flow,
+    // send them back to accept the invite
```

Comment:
I don't understand this comment in context of a render?

Reply:
It's just describing what `pendingInvite` is for